eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/EventStoreTransaction.java

```diff
@@ -39,23 +38,13 @@ public interface EventStoreTransaction {
     /**
      * Sources a {@link MessageStream} of type {@link EventMessage} based on the given {@code condition} that can be
      * used to rehydrate a model.
-     * <p>
-     * The given {@code context} is used to perform tasks at the right
-     * {@link org.axonframework.messaging.unitofwork.ProcessingLifecycle.Phase phases} and register resources throughout
-     * its lifecycle. All {@code condition} instances provided to source as part of the given {@code context} should be
-     * used to derive the {@link AppendCondition} typically used by the {@link #appendEvent(EventMessage)} method's
-     * implementation.
      *
      * @param condition The {@link SourcingCondition} used to retrieve the {@link MessageStream} containing the sequence
      *                  of events that can rehydrate a model.
-     * @param context   The {@link ProcessingContext} used to perform tasks at the right
-     *                  {@link org.axonframework.messaging.unitofwork.ProcessingLifecycle.Phase phases} and register
-     *                  resources throughout its lifecycle.
      * @return The {@link MessageStream} of type {@link EventMessage} containing to the event sequence complying to the
      * given {@code condition}.
      */
-    MessageStream<? extends EventMessage<?>> source(@Nonnull SourcingCondition condition,
-                                                    @Nonnull ProcessingContext context);
```

Comment:
Good change! :) 
@smcvb we're talking about that before, looks like not needed.